--- src/main/java/org/apache/maven/plugin/javadoc/AbstractJavadocMojo.java~	2014-09-15 21:47:58.000000000 +0200
+++ src/main/java/org/apache/maven/plugin/javadoc/AbstractJavadocMojo.java	2014-09-16 09:31:43.828203477 +0200
@@ -3344,10 +3344,6 @@
             throw new MavenReportException( "Unable to build the Maven project for the artifact:" + javadocArtifact,
                                             e );
         }
-        catch ( InvalidDependencyVersionException e )
-        {
-            throw new MavenReportException( "Unable to resolve artifact:" + javadocArtifact, e );
-        }
     }
 
     /**
